tests/unit/helpers/emptyModule.js

```diff
@@ -0,0 +1,6 @@
+// use to map any file or extension to an empty module
+module.exports = {
+  process() {
+    return ""
```

Comment:
what is this for? when is `emptyModule` used?

Reply:
i.e. for `.css` files which we don't want to import in jest